barman/backup.py

```diff
@@ -604,6 +604,13 @@ def _validate_incremental_backup_configs(self):
 
         :raises BackupException: if a required configuration is missing
         """
+        if self.config.backup_method != "postgres":
+            raise BackupException(
+                "incremental backups using '--incremental' only works when "
```

Comment:
I would change this error message to something like
```
Backup using the `--incremental` flag is available only for 'backup_method = postgres'.
Check barman's documentation for more help on this topic.
```

I'm not a huge wordsmith but in general, Is a bit more concise.
Error messages should begin with capital letters.

Reply:
Makes sense to me. I just fixed it ✌️